src/services/web-monitoring-api.js

```diff
@@ -77,6 +77,82 @@ export default class WebMonitoringApi {
             ));
     }
 
+    /**
+     * Add change information to the dictionary of insignificant changes in
+     * Google sheets.
+     *
+     * @param {Page}    page
+     * @param {Version} fromVersion
+     * @param {Version} toVersion
+     * @param {Object}  annotation
+     * @returns {Promise<boolean>}
+     */
+    addChangeToDictionary (page, fromVersion, toVersion, annotation) {
```

Comment:
👌 Got it. Though, I'd probably win in a 'Who's lazier?' contest :1st_place_medal: